README.md

```diff
@@ -1,13 +1,53 @@
-# Puppet 5 Modules for WSO2 Identity Server 5.7.0
+# Puppet Modules for WSO2 Identity Server
+
+This repository contains the Puppet modules for WSO2 Identity Server and the profiles related to Identity Server Analytics.
+
+## Prerequisites
+The Puppetmaster should run a `puppet agent` command on itself. As a result, the hostname `puppet` must resolve within itself. To do so, "puppet" should be added as a host in `etc/hosts` before the build script is executed.
```

Comment:
It would be great if we could use the "puppet apply" command instead of running a "puppet agent" in the master itself. If using the puppet apply command you won't need to add such configs to the /etc/hosts.
puppet apply -e "include is_master" --modulepath=/etc/puppet/code/environments/production/modules

Reply:
Fixed https://github.com/wso2/puppet-is/pull/72/commits/cc59a3c983d1f8501145fca0c0008b61fcf319e1